### ISSUE TITLE
Feature Request: Add DuckDuckGo Newsfeed And/Or Startpage Newsfeed

### ISSUE THREAD

**Issue by trymeouteh** [39aldo39/spaRSS-DecSync#6]
Google newsfeed is good but Google is not a good privacy tool, also Google is known to censor certain search results and could censor certain newsfeeds.

Please add alternatives such as DuckDuckGo, Startpage and others that have good privacy reputation and do not censor feeds.

**Comment by 39aldo39** [comment]
I think it would be good to have alternatives to the Google news feed, but after a quick search it doesn't look DuckDuckGo or Startpage have a news feed with RSS support?

**Comment by trymeouteh** [comment]
True. Are there any Google trackers when using Google News RSS feeds? I understand if you clicked on some of the articles and loaded them into a browser they could contain trackers but would the feed itself and the article in a RSS reader have trackers?

The reason I want to stay away from Google is because Google is known to hide search results and can alter feeds.

**Comment by 39aldo39** [comment]
The Google News feature will just take the RSS feed from a Google News topic. The usual privacy implications apply, i.e. Google knows when the feed is refreshed with some metadata (IP address etc.), but nothing more than that.

**Comment by trymeouteh** [comment]
> The Google News feature will just take the RSS feed from a Google News topic. The usual privacy implications apply, i.e. Google knows when the feed is refreshed with some metadata (IP address etc.), but nothing more than that.

OK but no trackers are snucked into the feed?
